features/facebookImport/src/components/belowChartButtons/belowChartButtons.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import "./belowChartButtons.css";
+
+const BelowChartButtons = ({
```

Comment:
Why _below_? If they absolutely _have_ to be below, I'd rather recommend to have a container with a button area at the bottom. This particular component probably shouldn't care about _where_ it's placed - how about just `ChartButtons` or `ChartButtonArea`?

Reply:
"New component belowChartButtons (Please help me give this a better name I'm clueless)", thanks :D

Reply:
Any of the two mentioned by @fhd are OK.

Reply:
I suppose I slightly prefer `ChartButtons`, but no strong opinion.

Reply:
When we go full atomic I think both will exist (Chartbutton -> ChartButtonArea). Now I went for ChartButtons